package.json

```diff
@@ -38,11 +38,22 @@
     "node": "^8.10.0 || ^10.13.0 || >=11.10.1"
   },
   "dependencies": {
-    "codependency": "^2.1.0",
     "json-stringify-safe": "^5.0.1",
     "lodash": "^4.17.15",
     "reconnect-core": "^1.3.0"
   },
+  "peerDependencies": {
+    "winston": "^3.x",
+    "winston-transport": "^4.3.0"
```

Comment:
considering we used to require 3.2.1+.... why are we now requiring an even higher version if we can probably work with both?

Reply:
Is this referring to https://github.com/rapid7/r7insight_node/pull/36#discussion_r823609461 or?

Reply:
nope. this sets a minimum version of 4.3.0 for winston-transport, right, whereas we worked with 3.2.1+ before?

Reply:
Before it worked with `*` I believe, but again same as the other version limiting, this is what we have verified works as it's what our devDependencies use